.github/workflows/torchserve-nightly-build.yml

```diff
@@ -6,28 +6,37 @@ on:
     - cron:  '15 11 * * *'
 jobs:
   nightly:
-    runs-on: ubuntu-18.04
+    # creates workflows for the 3 OS: ubuntu, macOS & windows
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-latest]
     steps:
-      - name: Setup Python 3.8
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.8
-          architecture: x64
+      - name: Setup Conda
+        uses: s-weigand/setup-conda@v1
+      - run: conda --version
+      - run: conda install -y conda-build anaconda-client
       - name: Checkout TorchServe
         uses: actions/checkout@v2
       - name: Install dependencies
         run: |
-          set -eux
-          pip install twine
-          sudo apt-get update
-          python ts_scripts/install_dependencies.py --cuda=cu102 --environment=dev
+          python ts_scripts/install_dependencies.py --environment=dev
           pip install -e .
       - name: Setup Java 11
         uses: actions/setup-java@v2
         with:
           distribution: 'zulu'
           java-version: '11'
-      - name: Push nightly
+      - name: Build PyPI & conda nightly binaries
+        run: python binaries/build.py --upload-conda-packages --nightly
```

Comment:
should we able to get rid of the `--upload-conda-packages` flag now

Reply:
Thanks @msaroufim . It wasn't auto-uploading but I completely missed removing this. Resolved it